Makefile

```diff
@@ -41,7 +41,7 @@ ifeq ($(HAZRUST),yes)
 .PHONY: jsoncdc.so
 jsoncdc.so:
 	cargo rustc $(FEATURES) --release -- $(LINK_FLAGS)
-	cp target/release/libjsoncdc.*[so\|dll] $@
+	cp target/release/libjsoncdc.[so\|dll] $@
```

Comment:
On Mac, it's `dylib` but I am honestly not sure this style of pattern is the way to go. In #80 there's a patch to this branch that uses GNU Make wildcards.